pkg/csi/service/wcp/controller.go

```diff
@@ -731,7 +748,16 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 			},
 		}
 	}
-
+	if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.PodVMOnStretchedSupervisor) {
+		// Create CNSVolumeInfo CR for the volume ID.
+		err = volumeInfoService.CreateVolumeInfoWithPolicyInfo(ctx, volumeInfo.VolumeID.Id,
+			storagePolicyID, storageClassName, vc.Config.Host)
+		if err != nil {
+			return nil, csifault.CSIInternalFault, logger.LogNewErrorCodef(log, codes.Internal,
+				"failed to store volumeID %q for vCenter %q in CNSVolumeInfo CR. Error: %+v",
```

Comment:
replace vcenter with or add storage policy id in log message

Reply:
Done